src/components/templates/Camp/components/RoomCard.tsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import { RoomAttendance } from "./RoomAttendance";
+import { formatMinute } from "utils/time";
+import { CampRoomData } from "types/CampRoomData";
+import { useSelector } from "hooks/useSelector";
+import { getCurrentEvent } from "utils/event";
+import "../../../templates/PartyMap/components/RoomCard/RoomCard.scss";
+
+interface PropsType {
+  room: CampRoomData;
+  attendance?: number;
+  onClick: () => void;
+}
+
+const RoomCard: React.FunctionComponent<PropsType> = ({
+  room,
+  attendance,
+  onClick,
+}) => {
+  const { venueEvents } = useSelector((state) => ({
+    venueEvents: state.firestore.ordered.venueEvents,
+  }));
```

Comment:
every time anything changes in redux, this causes a rerender as the callback returns a new object. Can be avoided using 
```
const venueEvents = useSelector((state) => (
    venueEvents: state.firestore.ordered.venueEvents
  ));
```